proto/xudrpc.proto

```diff
@@ -405,6 +413,30 @@ message ListPeersResponse {
   repeated Peer peers = 1 [json_name = "peers"];
 }
 
+message ListOrderHistoryRequest {
+  string order_id = 1 [json_name = "order_id"]; 
```

Comment:
It seems a bit weird for me to use this call to also look up single orders, that feels like it should be a separate rpc call. Instead maybe it can accept a `limit` like we do elsewhere, or maybe a parameter to only get orders that are less than X days old (can be a future PR though). Thoughts @kilrau?

Reply:
Without checking the code: ListOrderHistory is supposed to accept ONE `order_id` and lists all events like

Order status: filled/cancelled/open
Swapdeals:
Trades:
Log entries:

Nothing multiple orders..

Reply:
Ah ok my mistake, I'm trying to track too many things at once. I will review again with that in mind.